README.md

```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
 
 ![Image showing the parameter file creation options (none, new, and browse)](./images/undo-change-mapping-two.png)
 
+## Linked Template Support
+
+When a [linked template](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/linked-templates#linked-template) is referenced, schema and parameter validations are provided.
```

Comment:
It's not just parameters that are validated.  Some semantics of the linked template are validated, using the parameter values that were passed in.